x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/LogicalPlanBuilder.java

```diff
@@ -253,7 +254,45 @@ public Sequence visitSequence(SequenceContext ctx) {
                             found);
                 }
             }
-            queries.add(sequenceTerm);
+            // check runs
+            Token key = sequenceTermCtx.key;
+            if (key != null) {
+                String k = key.getText();
+                if (RUNS.equals(k) == false) {
+                    throw new ParsingException(source(key), "Unrecognized option [{}], expecting [{}]", k, RUNS);
+                }
+            }
+
+            int runs = 1;
```

Comment:
If this is limited to 100, why not using `byte`?

Reply:
Habit - since it's a local variable it doesn't make a difference.